partcad-cli/src/partcad_cli/click/commands/supply/quote.py

```
@@ -8,19 +8,30 @@
 import json
 import sys
 
+# ⬢ [Docker] ❯ pc supply quote --help
```

Ruff (0.8.2)

Comment contains ambiguous `❯` (HEAVY RIGHT-POINTING ANGLE QUOTATION MARK ORNAMENT). Did you mean `>` (GREATER-THAN SIGN)?

(RUF003)